modules/project-docs/pages/sdk-full-installation.adoc

```diff
@@ -130,20 +151,22 @@ extension=couchbase
 NOTE: on PHP version 7.2.0 and up, simply using the extension name is preferred.
 Previously, you might have used `couchbase.so` or `couchbase.dll`, depending on your platform.
 
-=== PHP Composer
+// TODO: unclear if this is still valid, there is no 4.0.0 package available at https://packagist.org/packages/couchbase/couchbase
```

Comment:
Unsure if this is still valid, or how we setup composer to work with SDK 4.0.0, please let me know if you have any suggestions @avsej .

Reply:
the packagist is hosting pure-PHP libraries. Ours is mixing PHP and C++, and we host it on pecl.php.net. The code that was hosted on the packagist was for 3.x branch, and was only implementing datastructures. Now this code  is integrated into 4.0.0, and we don't use packagist anymore.